src/main.rs

```diff
@@ -60,6 +60,12 @@ struct Opts {
         default = "false"
     )]
     hard_linebreaks: bool,
+    #[options(
+        no_short,
+        help = "Remove Obsidian style comments from exported file",
+        default = "false"
+    )]
+    remove_obsidian_comments: bool,
```

Comment:
I think I would like the CLI interface for this to be `--comments=[keep-unchanged|remove]`.

This would allow it to be expanded to  `--comments=[keep-unchanged|remove|as-html-comments]` in the future (and "as-html-comments" should probably become the default at that point).

Reply:
I like this implementation suggested and will get this changed over. However, I disagree when it comes to changing the default in the future. I think leaving the comments unchanged has been the way the tool works since the beginning and people would be expecting the tool to run this way potentially breaking workflows. I think it would be best practice to makes additions like this opt-in rather than opt-out.